### ISSUE TITLE
#2 Release a new version of theforeman-rubocop

### ISSUE THREAD

**Issue by archanaserver** [theforeman/theforeman-rubocop#13]
- [x] https://github.com/theforeman/theforeman-rubocop/pull/8
  - [x] https://github.com/theforeman/theforeman-rubocop/pull/9
- [x] https://github.com/theforeman/theforeman-rubocop/pull/11
- [x] https://github.com/theforeman/theforeman-rubocop/pull/24
- [x] https://github.com/theforeman/theforeman-rubocop/pull/25

For more details: https://community.theforeman.org/t/standardizing-rubocop-with-theforeman-rubocop/37239

**Comment by archanaserver** [comment]
@ekohl is there anything else that need to be done here?

**Comment by ekohl** [comment]
We need to update all the RuboCop version to the latest. There are new versions released by now (see open PRs).

And we don't need this dependency anymore and we should drop it
https://github.com/theforeman/theforeman-rubocop/blob/71cf8bff903e870f97bc995239b0fff53c3f5096/theforeman-rubocop.gemspec#L26

**Comment by ekohl** [comment]
Another thing I noticed while running RuboCop on Foreman is that I saw:
```
The following RuboCop extension libraries are installed but not loaded in config:
  * rubocop-capybara
  * rubocop-graphql

You can opt out of this message by adding the following to your config (see https://docs.rubocop.org/rubocop/extensions.html#extension-suggestions for more options):
  AllCops:
    SuggestExtensions: false
```

I think we should include those in `rules/rails.yml` as enabled extensions.

**Comment by archanaserver** [comment]
@ekohl raised PRs to fix it, #24 #25
